### ISSUE TITLE
`contexts.trace.data.route` getting rejected by Sentry

### ISSUE THREAD

**Issue by SystematicCZ** [getsentry/sentry-symfony#800]
### How do you use Sentry?

Sentry SaaS (sentry.io)

### SDK version

4.13.1

### Steps to reproduce

1. Trigger an event that sends a trace to Sentry.
2. Observe the trace details in the Sentry issue interface.
3. The `contexts.trace.data.route` field does not contain the actual route name.

### Expected result

contexts.trace.data.route should contain route name

### Actual result

![obrazek](https://github.com/getsentry/sentry-symfony/assets/53008590/24a8f5c6-30ea-4829-8095-edecd2797649)

```
{
http.flavor: 1.1,
 http.request.method: GET, 
 http.url: ...,
net.host.name: ..., 
net.host.port: 443, 
net.peer.ip: ..., 
route: None
}
```

**Comment by SystematicCZ** [comment]
No, we only use the `captureException()` function and we use Sentry relay. We do not alter the payload that is sent. My understanding is that these values are set by event listener from this bundle.

**Comment by cleptric** [comment]
Thanks, I'll take a look.

**Comment by markitosgv** [comment]
Same issue here, it seems something has changed

**Comment by cleptric** [comment]
I do confirm this issue, and it looks like something regarding ingestion has changed. Talking to the team to fix it. The good news is that the `route` tag is still applied correctly to all events, and tags are actually queryable/searchable while the context is not. So, while the error in the UI is annoying, it does not break anything.
